app/src/androidTest/kotlin/at/bitfire/davdroid/servicedetection/RefreshCollectionsWorkerTest.kt

```diff
@@ -88,6 +89,9 @@ class RefreshCollectionsWorkerTest {
 
     @Inject
     lateinit var db: AppDatabase
+    
+    @Inject
+    lateinit var collectionRepository: DavCollectionRepository
```

Comment:
I wonder whether we should extract the `Refresher` to a new file (and rename it to something like `CollectionListRefresher`).

Then we could properly inject our dependencies there. Also we would have even more separation between the worker (= adapter code) and the actual refresher (which is why it's already an own class).

Reply:
I will try to extract CollectionListRefresher now myself in a separate PR (and merge as soon as possible), please just merge it into this PR then.